assets/src/edit-story/components/library/panes/media/useSmallestImage.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Choose the source URL of the smallest available size image.
+ *
+ * @param {*} resource Image resource object.
+ * @return {string} Source URL of the smallest available size image.
+ */
+function useSmallestImage(resource) {
```

Comment:
Only hooks should start with `use...`, as that's what the linter uses to detect hooks.

Perhaps rename to `getThumbnailUrl` or something.

Reply:
This file also feels like this should like `app > meida > utils` as it acts the media resource object.

Reply:
Moved to app/media/utils/getThumbnailUrl.js